app/utils/Wallet/Wallet.js

```diff
@@ -0,0 +1,95 @@
+import { Token } from './Token'
```

Comment:
Name index files `index.js`. This way you can just import `from 'utils/Wallet'` instead of `from 'utils/Wallet/Wallet'`